champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/sword/BlockToss.java

```diff
@@ -220,7 +231,13 @@ public void updateCharge() {
                 chargeData.tickSound(player);
 
                 if (chargeData.getCharge() < 1) {
-                    chargeData.boulder.setSize(chargeData.boulder.getSize() + getSize(level) / 20);
+                    float chargeProgress = chargeData.getCharge(); // 0 to 1
+
+                    double newSize = startSize + chargeProgress * (size - startSize);
+                    chargeData.boulder.setSize(newSize);
+
+                    float newDisplacement = 0.05F + chargeProgress * (0.4F - 0.05F);
+                    chargeData.boulder.setDisplacement(newDisplacement);
```

Comment:
Correct me if I'm wrong, but this isn't actually working because the display entities' displacement is not being updated, but instead it's only queried when they are spawned.

Reply:
no it updates